_books/ion-1-1/src/text/e_expressions.md

```diff
@@ -0,0 +1,91 @@
+# E-expressions
+
+In Ion text, encoding expressions ([E-expressions](../macros/macros_by_example.md)) start with `(:`, immediately 
+followed by a macro reference, which must be one of:
+ * a base-10 integer macro address
+ * a macro name
+ * a qualified macro name consisting of a module name, double-colon (`::`), and the macro name. (See [Encoding modules](../modules/encoding_modules.md) for details about qualified macro references.)
```

Comment:
This can also be a qualified address (`foo::17`).

Reply:
I thought that was only in TDL, not for E-expressions. For E-Expressions, there is a single macro table that is defined by the encoding context, right?

Reply:
The macro table is a concatenation of the encoding modules' respective macro tables. You can use a qualified address to reference a macro by its address within a module provided that the module is in the encoding module sequence. (It looks like we don't show an example of this in the [_Encoding modules_](https://amazon-ion.github.io/ion-docs/books/ion-1-1/modules/encoding_modules.html) chapter, however. We only have examples for qualified names, not qualified addresses.)